### DISCUSSION TITLE
Color of header: Change back?

### DISCUSSION THREAD

**Discussion by vercte** [vercte/forumoji#27] [Polls]
I recently changed the header color, but 64lu suggested we change it back as it contrasts well with the emojis. Should we change it back?

**Comment by notwait** [comment]
Make it an option.

**Reply by vercte** [reply]
I don't see the necessity of making it an option, however. Why do you need options for a website where you pick emojis?

**Reply by Steve0Greatness** [reply]
I made a theme system in my own fork of this. It's surprisingly easy.

**Reply by stickfiregames** [reply]
created #28 for the addition of light and dark modes

**Comment by dinomaster20** [comment]
I'm kind of replying to [this](https://github.com/lopste/forumoji/discussions/27#discussioncomment-2814715). 

Maybe when you click theme, it brings up a box with options. Dark, Light - Red, Blue.
